Game/game_controller.rb

```diff
@@ -0,0 +1,73 @@
+require_relative 'game'
+require_relative 'author'
+
+class GameController
+  attr_accessor :games, :authors
+
+  def initialize
+    @games = []
+    @authors = []
+  end
+
+  def list_games
+    if @games.empty?
+      puts 'No game available!'
+    else
+      @games.each_with_index do |game, index|
+        print "#{index + 1} Multiplayer: #{game.multiplayer}, Last played at: #{game.last_played_at}, "
+        print "Author: #{game.author.first_name} #{game.author.last_name}"
+      end
+    end
+  end
+
+  def list_authors
+    if @authors.empty?
+      puts 'No author available!'
+    else
+      @authors.each_with_index { |author, index| puts "#{index + 1}: #{author.first_name} #{author.last_name}" }
+    end
+  end

```

Comment:
You're to work on the game and label only, list_authors method should not be part of your implementation remove it.